templates/rhdm71-prod-kieserver.yaml

```diff
@@ -0,0 +1,669 @@
+  required: false
+- displayName: Name of the internal Maven service's URL path
+  description: The URL path for an optional internal maven service, where it can be reached, to allow service lookups (for maven repo usage), if required
+  name: INTERNAL_MAVEN_REPO_PATH
+  example: "/maven2/"
+  required: false
+- displayName: Username for the internal Maven service
+  description: Username to access the internal Maven service.
+  name: INTERNAL_MAVEN_USERNAME
+  example: "mavenUser"
+  required: false
+- displayName: Password for the internal Maven service
+  description: Password to access the internal Maven service.
+  name: INTERNAL_MAVEN_PASSWORD
+  example: "maven1!"
+  required: false
+- displayName: KIE Admin User
+  description: KIE administrator username
+  name: KIE_ADMIN_USER
+  value: adminUser
+  required: false
+- displayName: KIE Admin Password
+  description: KIE administrator password
+  name: KIE_ADMIN_PWD
+  from: "[a-zA-Z]{6}[0-9]{1}!"
+  generate: expression
+  required: false
+- displayName: KIE Server User
+  description: KIE server username (Sets the org.kie.server.user system property)
+  name: KIE_SERVER_USER
+  value: executionUser
+  required: false
+- displayName: KIE Server Password
+  description: KIE server password (Sets the org.kie.server.pwd system property)
+  name: KIE_SERVER_PWD
+  from: "[a-zA-Z]{6}[0-9]{1}!"
+  generate: expression
+  required: false
+- displayName: ImageStream Namespace
+  description: Namespace in which the ImageStreams for Red Hat Middleware images are
+    installed. These ImageStreams are normally installed in the openshift namespace.
+    You should only need to modify this if you've installed the ImageStreams in a
+    different namespace/project.
+  name: IMAGE_STREAM_NAMESPACE
+  value: openshift
+  required: true
+- displayName: KIE Server ImageStream Name
+  description: The name of the image stream to use for KIE server. Default is "rhdm71-kieserver-openshift".
+  name: KIE_SERVER_IMAGE_STREAM_NAME
+  value: "rhdm71-kieserver-openshift"
+  required: true
+- displayName: ImageStream Tag
+  description: A named pointer to an image in an image stream. Default is "1.0".
+  name: IMAGE_STREAM_TAG
+  value: "1.0"
+  required: true
+- displayName: KIE Server Controller User
+  description: KIE server controller user name. The Decision Server uses this user account to log on to Decision 
+    Central or a controller. (Sets the org.kie.server.controller.user system property).
+  name: KIE_SERVER_CONTROLLER_USER
+  value: controllerUser
+  required: false
+- displayName: KIE Server Controller Password
+  description: KIE server controller password (sets the org.kie.server.controller.pwd system property).
+  name: KIE_SERVER_CONTROLLER_PWD
+  from: "[a-zA-Z]{6}[0-9]{1}!"
+  generate: expression
+  required: false
+- displayName: KIE server controller protocol
+  description: KIE server controller service. To use Decision Central that is hosted on the same OpenShift environment
```

Comment:
The description seems to be related to service property below, not this KIE_SERVER_CONTROLLER_PROTOCOL